### PR TITLE
Inserted a dimmer with the loading message in the form when the statu…

### PR DESCRIPTION
**Pull request by damiDev8**
The dimmer with the loading message has been inserted in the rename and workflow change forms. To do this, two props have been set up in the ModalForm to understand if it is loading and the loading message to show. a problem with modal overflow is fixed

### CODE REVIEW

File: locales/it/LC_MESSAGES/volto.po [hunk 1]
```diff
@@ -954,6 +954,10 @@ msgstr "Elemento/i aggiornati."
 msgid "Item(s) pasted."
 msgstr "Elemento/i incollato/i."
 
+#: components/manage/Contents/Contents
+msgid "Item(s) workflow has been updated."
+msgstr ""
```
Comment (giuliaghisini) [inline locales/it/LC_MESSAGES/volto.po:959]: @damiDev8 add italian translation

File: locales/it/LC_MESSAGES/volto.po [hunk 2]
```diff
@@ -1021,6 +1025,10 @@ msgstr "Link a"
 msgid "Listing"
 msgstr "Elenco"
 
+#: components/manage/Form/ModalForm
+msgid "Loading"
+msgstr ""
```
Comment (giuliaghisini) [inline locales/it/LC_MESSAGES/volto.po:1030]: @damiDev8 add italian translation

File: locales/it/LC_MESSAGES/volto.po [hunk 3]
```diff
@@ -1425,6 +1433,10 @@ msgstr "Rimuovi"
 msgid "Rename"
 msgstr "Rinomina"
 
+#: components/manage/Contents/ContentsRenameModal
+msgid "Rename Items Loading Message"
+msgstr ""
```
Comment (giuliaghisini) [inline locales/it/LC_MESSAGES/volto.po:1438]: @damiDev8 add italian translation

File: locales/it/LC_MESSAGES/volto.po [hunk 4]
```diff
@@ -2094,6 +2106,10 @@ msgstr "Quando questa data sarà raggiunta, il contenuto non sarà più visibile
 msgid "Who"
 msgstr "Chi"
 
+#: components/manage/Contents/ContentsWorkflowModal
+msgid "Workflow Change Loading Message"
+msgstr ""
```
Comment (giuliaghisini) [inline locales/it/LC_MESSAGES/volto.po:2111]: @damiDev8 add italian translation

**Review by giuliaghisini — CHANGES_REQUESTED** [review]

File: src/components/manage/Contents/Contents.jsx [hunk 5]
```diff
@@ -151,6 +151,10 @@ const messages = defineMessages({
     id: 'Item(s) pasted.',
     defaultMessage: 'Item(s) pasted.',
   },
+  messageWorkflowUpdate: {
+    id: 'Item(s) workflow has been updated.',
```
Comment (tiberiuichim) [inline src/components/manage/Contents/Contents.jsx:155]: The modal dialog says "Change state", the Contents toolbar says "Change state", we should keep the wording. Editors may not exactly know what "workflow" means.

**Review by tiberiuichim — CHANGES_REQUESTED** [review]

**Review by tiberiuichim — APPROVED** [review]

**Comment by damiDev8** [comment]
hi, now the request should be ready to be merged @sneridagh

**Review by sneridagh — APPROVED** [review]

**Pull request merged**